eclass/kernel-install.eclass

```diff
@@ -260,6 +357,13 @@ kernel-install_test() {
 
 	local qemu_arch=$(kernel-install_get_qemu_arch)
 
+	local uki="${image%/*}/uki.efi"
+	# If we are testing a gpkg built with USE=generic-uki, we only have an
+	# uki.efi, extract the kernel image.
+	if [[ -f ${uki} && ! -f ${image} ]]; then
+		objcopy -O binary -j.linux "${uki}" "${image}" || die "Failed to extract kernel image"
```

Comment:
Respect `${OBJCOPY}` from the environment (use the toolchain-funcs helpers).

Reply:
Thanks, Fixed!